flutter_local_notifications_windows/lib/src/details/notification_input.dart

```diff
@@ -0,0 +1,116 @@
+import 'package:xml/xml.dart';
+
+/// The type of a [WindowsInput].
+enum WindowsInputType {
+  /// A text input.
+  text,
+
+  /// A multiple choice input.
+  selection,
+}
+
+/// A text or multiple choice input element in a Windows notification.
+abstract class WindowsInput {
+  /// Creates an input field in a notification.
+  const WindowsInput({
+    required this.id,
+    required this.type,
+    this.title,
+  });
+
+  /// A unique ID for this input.
+  ///
+  /// Can be used by buttons to be placed next to this input.
+  final String id;
+
+  /// The type of this input.
+  final WindowsInputType type;
+
+  /// The title of this input.
+  final String? title;
+
+  /// Serializes this input to XML.
+  void toXml(XmlBuilder builder);
```

Comment:
Can you move this out so it's not part of the public API surface? Could you also see if this could be renamed? It doesn't return a result and methods named `to<...>` conventionally return a result due to how they're named